src/state/queries/feed.ts

```diff
@@ -199,10 +206,14 @@ export function useSearchPopularFeedsMutation() {
   })
 }
 
-const FOLLOWING_FEED_STUB: FeedSourceInfo = {
+/**
+ * The following feed, with fallbacks to Discover
+ */
+const HOME_FEED_STUB: FeedSourceInfo = {
+  isPrimaryAlgorithm: false,
   type: 'feed',
   displayName: 'Following',
-  uri: '',
+  uri: 'home',
```

Comment:
All feeds are now driven by the `uri` value. For following feeds, it's `following` (unused atm) or `home` (which is Following + fallbacks). These two values have special handling elsewhere in the codebase to map to `getTimeline` instead of `getFeed`.